lib/PuppeteerSharp.Tests/TestConstants.cs

```diff
@@ -39,6 +41,17 @@ public static class TestConstants
             LogProcess = true
         };
 
+        public static LaunchOptions BrowserWithExtensionOptions() => new LaunchOptions
```

Comment:
This can be a property

Reply:
It's similar to the `DefaultBrowserOptions` method

Reply:
Fair enough :joy: